### ISSUE TITLE
Focus on the feed panel after an action

### ISSUE THREAD

**Issue by naku** [sismics/reader#29]
After doing an action such as executing a search, clicking on "all" or "unread" buttons, it's impossible to use the arrow keys or pageup/down to scroll (I have to click on the feed panel first).

The keyboard focus should be set immediately on the feed panel after such an action to facilitate the navigation.
